src/bitcoin-tx.cpp

```diff
@@ -468,7 +468,11 @@ static void MutateTxSign(CMutableTransaction& tx, const std::string& strInput)
     }
 
     const CKeyStore& keystore = tempKeystore;
-    const PrecomputedTransactionData txdata(mergedTx);
+    // We don't support v5 transactions via this API yet.
+    if (mergedTx.nVersion < ZIP225_TX_VERSION) {
```

Comment:
Blocking: Why isn't this `>= ZIP225_TX_VERSION`?

Reply:
Whoops, that indeed should be the other way around (there's another spot that is this way around and I probably got them mixed up at 2am on Saturday).

Reply:
This is still outstanding.